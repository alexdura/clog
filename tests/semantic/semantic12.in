Arc(1, 2).
Arc(2, 3).

Path(x, y) :- Arc(x, y).
Path(x, y) :- inline Arc(x, t), Path(t, y).

Circular(x) :- inline Path(x, y), inline Path(y, x).