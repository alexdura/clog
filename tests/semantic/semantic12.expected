Literal inline Path(x, y) can't be inlined. The predicate is defined in a recursive rule.
Literal inline Path(y, x) can't be inlined. The predicate is defined in a recursive rule.
